packages/local_auth/local_auth_platform_interface/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  intl: ^0.17.0
+  intl: ">=0.17.0 <0.19.0"
```

Comment:
This `intl` dependency is actually cruft from before a refactoring, so rather than updating the range we should just remove it entirely.